apps/antalmanac/tests/download-ics.test.ts

```diff
@@ -2,68 +2,51 @@ import { EventAttributes } from 'ics';
 import type { Schedule } from '@packages/antalmanac-types';
 import { describe, test, expect } from 'vitest';
 import { getEventsFromCourses } from '$lib/download';
+import { CourseEvent, CustomEvent } from '$components/Calendar/CourseCalendarEvent';
 
 describe('download-ics', () => {
     test('converts schedule courses to events for the ics library', () => {
-        const courses: Schedule['courses'] = [
+        const courses: (CourseEvent | CustomEvent)[] = [
```

Comment:
We have a type that represents a union of all our event types.

```suggestion
        const courses: CalendarEvent[] = [
```